### PR TITLE
PROC-1336: Refactor incognito naming

### PR DESCRIPTION
**Pull request by zacharygoodwin**

### CODE REVIEW

File: proctor-common/src/main/java/com/indeed/proctor/common/model/ConsumableTestDefinition.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public class ConsumableTestDefinition {
     @Nullable private TestDependency dependsOn;
 
     private boolean isDynamic = false;
-    private boolean incognito = false;
+    private boolean evaluteForIncognitoUsers = false;
```
Comment (vanhaxl) [inline proctor-common/src/main/java/com/indeed/proctor/common/model/ConsumableTestDefinition.java:36]: Is it "evaluateForIncognitoUsers"?

Comment (zacharygoodwin) [reply proctor-common/src/main/java/com/indeed/proctor/common/model/ConsumableTestDefinition.java:36]: Good catch: fixed typo.

File: proctor-common/src/main/java/com/indeed/proctor/common/Proctor.java [hunk 2]
```diff
@@ -331,12 +331,11 @@ public ProctorResult determineTestGroups(
                 ProctorUtils.convertToValueExpressionMap(
                         RuleEvaluator.EXPRESSION_FACTORY, inputContext);
 
-        final boolean evaluateAnonymousTestsOnly = getAnonymousEnabled(inputContext);
-
         for (final String testName : filteredEvaluationOrder) {
             final TestChooser<?> testChooser = testChoosers.get(testName);
             final String identifier;
-            if (evaluateAnonymousTestsOnly && !testChooser.getTestDefinition().getIncognito()) {
+            if (getIncognitoEnabled(inputContext)
```
Comment (vanhaxl) [inline proctor-common/src/main/java/com/indeed/proctor/common/Proctor.java:337]: NIT, Is the method name `isIncognitoEnabled(inputContext)` better?

Comment (zacharygoodwin) [reply proctor-common/src/main/java/com/indeed/proctor/common/Proctor.java]: Renamed

**Review by vanhaxl — APPROVED** [review]

**Review by stevenchen-indeed — APPROVED** [review]

**Pull request merged**